### ISSUE TITLE
Opening .md files for editing shouldn't make them read only

### ISSUE THREAD

**Issue by subsoap** [defold/editor2-issues#808]
.md and probably other text file types need to be able to be edited from being opened.

**Comment by subsoap** [comment]
.txt files also need to be editable in the editor when opened.

**Comment by AGulev** [comment]
@subsoap Looks like not issue anymore?

**Comment by subsoap** [comment]
No, they still open in a viewer rather than editor, and the viewer for them still often breaks.

![test](https://user-images.githubusercontent.com/409170/49343152-39d9f200-f619-11e8-9899-4901e88e6e61.png)

This file should have content but it opens to nothing.

**Comment by erikangelin-king** [comment]
@subsoap Is there anything in the logs that could relate to the empty/broken .md viewer?
